concourse/scripts/val_from_yaml.rb

```diff
@@ -51,7 +51,9 @@ def [](key)
   val = property_tree[key]
   abort "Unable to find key: #{key}" if val.nil?
 
-  if val.is_a?(Array) || val.is_a?(Hash)
+  if val.is_a?(Array)
+    puts val.join(',')
+  elsif val.is_a?(Hash)
```

Comment:
Are you sure that nothing is using the previous behaviour which outputs an Array as YAML?

Reply:
I find this change dangerous too, you should come up with a backward-compatible solution for this.